README.md

```diff
@@ -7,13 +12,13 @@
 4| [The Great Alexa Hackathon](https://content.techgig.com/registrations-open-for-alexa-skills-hackathon-at-code-gladiators-2019/articleshow/68922361.cms) | July 2019 | HackerEarth | Submitted, Failed |
 5| [HackIIITD](https://hackiiitd.tech/) | 23-24 August 2019 | IIITD | 4th Place Win |
 6| [Hack Your Reality](https://hackyourreality.hackerearth.com/) | August 2019 | HackerEarth | Top 12 |
-7| [NASA space apps challenge qualifiers](https://2019.spaceappschallenge.org/locations/new-delhi/) | September 2019 | Galgotias, Greater Noida | Submitted, Failed |
+7| [NASA space apps challenge qualifiers](https://2019.spaceappschallenge.org/locations/new-delhi/) | September 2019 | Galgotias, Greater Noida | 1st Place Win |
```

Comment:
Remove this, I didn't win this one

Reply:
Got the point, but I won = we won
We're a team, right?

Reply:
You get it wrong everytime. 
Denied

Reply:
@midopooler, I didn't expect this from you
You're a great guy, but you're doing this.
I don't know why?